model/model.go

```diff
@@ -384,9 +384,19 @@ type ViewInfo struct {
 	Algorithm   ViewAlgorithm      `json:"view_algorithm"`
 	Definer     *auth.UserIdentity `json:"view_definer"`
 	Security    ViewSecurity       `json:"view_security"`
-	SelectStmt  string             `json:"view_select"`
+	SelectStmt  []string           `json:"view_select"`
 	CheckOption ViewCheckOption    `json:"view_checkoption"`
-	Cols        []CIStr            `json:"view_cols"`
```

Comment:
why removing `Cols`?

Reply:
before select statement contains wildcard, I need this attr to store some info, I raise this pr to expand select statement wildcard, so this attr no more needed.

Reply:
OK. Test failed, seems you should modify some tests in the tidb repo: https://circleci.com/gh/AndrewDi/parser/73?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Reply:
PR https://github.com/pingcap/tidb/pull/9328 has fix this test.